### PR TITLE
Update stories.json

### PR DESCRIPTION
**Pull request by Oshratn**
10 additional stories

### CODE REVIEW

File: stories.json [hunk 1]
```diff
@@ -33,5 +33,60 @@
     "permalink": "good-times",
     "text": "We have a holistic view of people. Work life, a family life,  includeanything life are not silos. People have one life. Organizations need to consider that. It is less about partying and onsites (even if they mean a lot to me <3), it is more about having a purpose in your daily work (no bullshit jobs) and the `go` to self-organize. It is about no boundaries to thinking and getting the information you need to make decisions (transparency) and decide as an adult person what is best for you. It is about the flexibility to integrate all elements of life into your day.",
     "background": "northernlights"
+  },
+  {
+    "permalink": "long-run",
+    "text": "We sold a few companies. We are having fun. How do we make it stay that way, how do we make this work?",
+    "background": "gradient"
+  },
```
Comment (snizhana-dynnyk) [inline stories.json:41]: I think this is not a story yet.. Probably it was just an idea of a future story 🤷‍♀️

File: stories.json [hunk 2]
```diff
@@ -33,5 +33,60 @@
...
+  "text": "It's all about getting the job done. Not when and where - just get it done. Our culture allows for that because we trust each other.",
+  "background": "northernlights"
+  },
+  {
+  "permalink": "acceptable-customers",
+  "text": "We said no to Raytheon (a weapons manufacturer) when they came by our booth and were ready to come back later to sign a $1M contract.",
+  "background": "bokehballs.002"
+  },
+  {
+  "permalink": "common-sense",
```
Comment (snizhana-dynnyk) [inline stories.json:68]: there is already a story with the same 'permalink' - https://culture.giantswarm.io/#common-sense.
This field should be unique, maybe 'common-sense2' ahaha

File: stories.json (hunk 1 above)
Comment (Oshratn) [reply stories.json]: Removed it for now.

File: stories.json (hunk 2 above)
Comment (Oshratn) [reply stories.json]: changed the permalink according to your suggestion

**Comment by Oshratn** [comment]
> I would rewrite the last few stories, but maybe we can publish like this and edit later (or ask people to edit).

I did some rewriting and editing, but sometimes I was intentional in leaving it the way it was written by the contributor so that on a whole it feels authentic and diverse and written by someone.

**Pull request merged**